app/translations/ru.js

```diff
@@ -0,0 +1,10 @@
+export const messages = {
+  app_title: "TronWatch",
+  name: "Название",
+  wallets: "Кошельки",
+  tokens: "Токены",
+  vote: "Голоса",
+  contacts: "Смарт-Контакты",
```

Comment:
```suggestion
  contacts: "контакты",
```

This is for _Contacts_ (as in a contact in an address book), not _Contracts_.